x-pack/plugins/enterprise_search/public/applications/app_search/components/crawler/components/add_domain/validation_state_icon.test.tsx

```diff
@@ -20,6 +20,12 @@ describe('ValidationStateIcon', () => {
     expect(wrapper.find(EuiIcon).prop('color')).toEqual('success');
   });
 
+  it('shows an alert icon when invalid', () => {
```

Comment:
Should this have been

```suggestion
  it('shows a warning icon when invalid', () => {
```

?